docupload/models.py

```diff
@@ -1,13 +1,16 @@
+from django.conf import settings
 from django.db import models
 
 
-# Create your models here.
-
 class Documentation(models.Model):
     '''
     Documentation: Model class which holds details of uploaded documentation file
     '''
 
     name = models.CharField(max_length=200)
-    doc_file = models.FilePathField('../static/docs/')
```

Comment:
Chill. We'll deal with docupload stuff later. Abhi focus on hacker_news and account